jsonrpcbase/main.py

```diff
@@ -184,7 +184,10 @@ def _invalid_params_response(self, err, id=None, jsonrpc=DEFAULT_JSONRPC):
         resp = {'id': id}
         self._fill_ver(jsonrpc, resp)
         resp['error'] = {
-            'message': err.message,
+            'message': 'Invalid params',
+            'data': {
+                'details': err.message,
```

Comment:
.message seemed good enough to me for now.

If in the future we decide we want more details on jsonschema errors, we can expand this 'data' dict here. The instance of ValidationError has a lot of additional information that we could potentially use.